native/src/main/java/tech/sourced/babelfish/JsonASTVisitor.java

```diff
@@ -1171,6 +1244,19 @@ public int visit(IASTDeclSpecifier node) {
         return PROCESS_SKIP;
     }
 
+    @Override
+    public int visit(IASTDeclSpecifier node)
+    {
+        if ((node instanceof IASTSimpleDeclSpecifier ||
```

Comment:
Can you please add a comment as to why this special case is here?

Reply:
Done in next commit.